controllers/servers/host_definer/storage_manager/host.py

```diff
@@ -0,0 +1,42 @@
+from controllers.common.csi_logger import get_stdout_logger
+from controllers.servers.host_definer.common import settings
+from controllers.servers.host_definer.common.types import VerifyHostResponse
+
+logger = get_stdout_logger()
+
+
+class StorageHostManager:
+    def __init__(self):
+        pass
+
+    def define_host(self, host_request):
```

Comment:
```suggestion
    def define_host(self, request):
```
request [is sufficient](https://github.com/IBM/ibm-block-csi-driver/blob/develop/controllers/servers/csi/csi_controller_server.py#L30)

Reply:
yes please